packages/csslsrs/README.md

```diff
@@ -1,3 +1,28 @@
-# CSSlsrs - WASM CSS Language Service 
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/web-lsp/weblsp/refs/heads/main/.github/assets/CSSlsrs_logo_dark.svg" />
+  <img alt="WEBlsp logo" src="https://raw.githubusercontent.com/web-lsp/weblsp/refs/heads/main/.github/assets/CSSlsrs_logo_light.svg" />
+</picture>
 
-Work in progress.
\ No newline at end of file
+A CSS Language Service made with Rust, for the JavaScript/TypeScript ecosystem.
+
+(Work In Progress) ~~It provides full-featured language support for HTML and CSS, including code completion, diagnostics, hover, and more.~~ Compliant with the [Language Server Protocol](https://microsoft.github.io/language-server-protocol/), CSSlsrs can be integrated into Language Servers written in Rust or JavaScript/TypeScript (via WebAssembly). Thanks to [Biome](https://biomejs.dev/)'s parser, we deliver a fast and efficient language service, designed to improve your productivity when working with CSS.
```

Comment:
This description in general seems superfluous. Since this package is for "end-users" (as in, people in JS using it for support), it'd probably be better to do a short high-level summary and let the crate / main README / marketing website say the rest.

Most notably, the last sentence does not strike me as necessary